components/automate-ui/src/app/entities/projects/project.requests.ts

```diff
@@ -28,10 +28,10 @@ export class ProjectRequests {
     return this.http.get<ProjectSuccessPayload>(`${env.iam_url}/projects/${id}`);
   }
 
-  public createProject(id: string, name: string): Observable<ProjectSuccessPayload> {
+  public createProject(id: string, name: string, skip: boolean): Observable<ProjectSuccessPayload> {
     return this.http.post<ProjectSuccessPayload>(
       `${env.iam_url}/projects`,
-      { id, name });
+      { id: id, name: name, skip_policies: skip });
```

Comment:
This would be more idiomatic. (`id` is shorthand for `id: id`, etc.)
```suggestion
      { id, name, skip_policies });
```
For that to work, though, change `skip` in L31 to skip_policies.